<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>简单ComboBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnInitItems">
              <property name="text">
               <string>初始化列表</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClearItems">
              <property name="text">
               <string>清除列表</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxEdit">
              <property name="text">
               <string>可编辑</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxItem">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>item1</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/icons/aim.ico</normaloff>:/images/icons/aim.ico</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item2</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/icons/aim.ico</normaloff>:/images/icons/aim.ico</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>item3</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/icons/aim.ico</normaloff>:/images/icons/aim.ico</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>有用户数据的ComoBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btnInitCitys">
           <property name="text">
            <string>初始化城市+区号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCity">
           <item>
            <property name="text">
             <string>北京市</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/iimages/icons/UNIT.ICO</normaloff>:/iimages/icons/UNIT.ICO</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>上海市</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/iimages/icons/UNIT.ICO</normaloff>:/iimages/icons/UNIT.ICO</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>苏州市</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/iimages/icons/UNIT.ICO</normaloff>:/iimages/icons/UNIT.ICO</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>QPlainTextEdit(具有标准快捷菜单)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAddToBox">
           <property name="text">
            <string>文本框内容添加到ComboxBox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClearContent">
           <property name="text">
            <string>清除文本内容</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxRead">
           <property name="text">
            <string>Read Only</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
